e2store/src/era.rs

```diff
@@ -109,13 +109,19 @@ impl Era {
     }
 
     fn get_block_slot_indexes(slot_index_block_entry: &SlotIndexBlockEntry) -> Vec<u64> {
+        if slot_index_block_entry.slot_index.indices.is_empty() {
+            return vec![];
+        }
+        // minus 8 because the first 8 bytes are the starting slot
```

Comment:
nit: it's minus 8 bytes for the first "entry", not "slot", right?
That should be `VersionEntry`, if I'm not wrong. Should we add something like `SERIALIZED_SIZE` to it (like we have for `Header`)?

Reply:
![image](https://github.com/user-attachments/assets/17ac73f9-35d7-4681-8987-e38ffb69041d)
`starting_slot` is 8 bytes, 

This has nothing to do with an entry

Reply:
![image](https://github.com/user-attachments/assets/18d47eda-d1a4-464e-a9ae-883678b29005)
Reading it again I am so confused, but I know it is 8 bytes